buildkite/resource_organization_settings_test.go

```diff
@@ -86,29 +86,12 @@ func TestAccOrganizationSettings_import(t *testing.T) {
 	})
 }
 
-func TestAccOrganizationSettings_disappears(t *testing.T) {
```

Comment:
Why is this removed? As with most of these migrations, we shouldn't need to change the acceptance tests

Reply:
We also need a test against the new resource name buildkite_organization

Reply:
Yeah, this can technically be removed if you’re checking for the removal on the Create test, but given that only tests that the resource is added you’ll need to check it has been removed too.